spec/features/hiring_staff_can_copy_a_vacancy_spec.rb

```diff
@@ -0,0 +1,116 @@
+require 'rails_helper'
+RSpec.feature 'Copying a vacancy' do
+  let(:school) { create(:school) }
+
+  before(:each) do
+    stub_hiring_staff_auth(urn: school.urn)
+  end
+
+  scenario 'hiring staff can see a Duplicate link on published and pending jobs' do
+    create(:vacancy, school: school, status: 'published')
+    create(:vacancy,
+           school: school,
+           status: 'published',
```

Comment:
[You can use FactoryBot traits for building objects with a particular and repeatable state](https://robots.thoughtbot.com/remove-duplication-with-factorygirls-traits), this enables us to change the definition of what a status means to the system centrally, without having to update every test. You may be able to lose both the `expires_on` and `publish_on` field as a result throughout this file.

Reply:
I think I used non-FactoryBot here as most of the existing feature tests don't use it and I assumed it was the style 😳 🤷‍♀️   Probably worth logging a refactor ticket to get them all using the same structure.